.github/workflows/review_pull_request.yml

```diff
@@ -47,6 +47,12 @@ jobs:
 
       # template-compose
 
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
```

Comment:
Should we use the latest version?
`actions/setup-java@v3`

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/508/commits/799ed77f8e6da386ee2aead09e7a3681c793acd8